### PR TITLE
(fix) update ui in billing for consistency.

### PR DESCRIPTION
**Pull request by munania**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

**Comment by munania** [comment]
OLD UI
OLD UI

**Comment by munania** [comment]
NEW UI
UI CHANGE

**Comment by munania** [comment]
Tablet view
tablet view ui

File: packages/esm-billing-app/src/billing-form/billing-form.component.tsx [hunk 1]
```diff
@@ -192,108 +193,110 @@ const BillingForm: React.FC<BillingFormProps> = ({ closeWorkspace }) => {
...
+              ))}
+            </ul>
+          </div>
 
-      {/* <NumberInput id="carbon-number" min={0} max={100} value={50} ref={numberRef}
-      onChange={(e)=> alert((numberRef.current as HTMLInputElement).value)} 
-      className="testingNumberInput" label="NumberInput label" helperText="Optional helper text." invalidText="Number is not valid" /> */}
+          {/* <NumberInput id="carbon-number" min={0} max={100} value={50} ref={numberRef}
+        onChange={(e)=> alert((numberRef.current as HTMLInputElement).value)} 
+        className="testingNumberInput" label="NumberInput label" helperText="Optional helper text." invalidText="Number is not valid" /> */}
```
Comment (donaldkibet) [inline packages/esm-billing-app/src/billing-form/billing-form.component.tsx:240]: Just do a clean up for this commented out logic

**Review by donaldkibet — APPROVED** [review]
Thanks @munania please follow up with PR to remove dead and commented code 🤝

**Pull request merged**